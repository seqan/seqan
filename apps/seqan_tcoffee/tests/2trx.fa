>1thx
AEQPVLVYFWASWCGPCQLMSPLINLAANTYSDRLKVVKLEIDPNPTTVKKYKVEGVPALRLVKGEQIL
DSTEGVISKDKLLSFLDTHLN
>1grx
MQTVIFGRSGCPYSVRAKDLAEKLSNERDDFQYQYVDIRAEGITKEDLQQKAGKPVETVPQIFVDQQHI
GGYTDFAAWVKENLDA
>1erv
AGDKLVVVDFSATWCGPCKMIKPFFHSLSEKYSNVIFLEVDVDDCQDVASECEVKSMPTFQFFKKGQKV
GEFSGANKEKLEATINELV
>2trcP
KVTTIVVNIYEDGVRGCDALNSSLECLAAEYPMVKFCKIRASNTGAGDRFSSDVLPTLLVYKGGELISN
FISVAEQFAEDFFAADVESFLNEYGLLPER
