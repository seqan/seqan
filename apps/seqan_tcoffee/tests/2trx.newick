(1thx,(1grx,(1erv,2trcP)))
