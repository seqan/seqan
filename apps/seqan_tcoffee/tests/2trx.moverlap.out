>1thx
-AEQPVLVYFWASWCGPCQLMSPLINLAANTYSDRLKVVKLEI-------DPNPTTVKKY
KVEGVPALRLVKGEQILD---STEGVISKDKLLS----FLDTHLN--------
>1grx
----MQTVIFGRSGCPYSVRAKDLAEKLSNERDDFQ-YQYVDIRAEGITKEDLQQKAGK-
PVETVP--QIFVDQQHIGGYTDFAAW-VKENLDA-------------------
>1erv
AGDKLVVVDFSATWCGPCKMIKPFFHSLSEKYSNVI-FLEVDV-------DDCQDVASEC
EVKSMPTFQFFKKGQKVG---EFSGA-NKEKLEA----TINELV---------
>2trcP
-KVTTIVVNIYEDGVRGCDALNSSLECLAAEYPMVK-FCKIRA-------SN-TGAGDRF
SSDVLPTLLVYKGGELIS---NFISV-AEQFAEDFFAADVESFLNEYGLLPER
